docssrc/src/commandtrees.md

```diff
@@ -80,6 +80,108 @@ In this example, we have our normal `/sayhi` command using the `executes()` meth
 
 </div>
 
+### uses nested arguments to reduce indentation levels
+
+sometimes we will need such a CommandTree:
+
+<div class="multi-pre">
+
+```java,Java
+new CommandTree("example")
+    .then(new LiteralArgument("arg1")
+    .then(new StringArgument("arg2")
+        .then(new StringArgument("arg3")
+            .then(new DoubleArgument("arg4", 0)
+                .then(new StringArgument("arg5"))
+                    .executes(...)))))
+    .register();
+```
+
+```kotlin,Kotlin
+CommandTree("example")
+    .then(LiteralArgument("arg1")
+        .then(StringArgument("arg2")
+            .then(StringArgument("arg3")
+                    .then(DoubleArgument("arg4", 0)
+                        .then(StringArgument("arg5")
+                            .executes { ... })))))
+    .register()
+```
+
+```kotlin,Kotlin DSL
```

Comment:
There should be an `_` instead of a space here (same for the other Kotlin DSL block)

```suggestion
```kotlin,Kotlin_DSL
```